Mathlib/SetTheory/Ordinal/Principal.lean

```diff
@@ -134,7 +134,7 @@ theorem principal_add_of_le_one {o : Ordinal} (ho : o ≤ 1) : Principal (· + 
   · exact principal_zero
   · exact principal_add_one
 
-theorem principal_add_isLimit {o : Ordinal} (ho₁ : 1 < o) (ho : Principal (· + ·) o) :
+theorem isLimit_of_principal_add {o : Ordinal} (ho₁ : 1 < o) (ho : Principal (· + ·) o) :
```

Comment:
Since `o` is implicit anyway, this might be more convenient as
```suggestion
theorem IsLimit.of_principal_add {o : Ordinal} (ho₁ : 1 < o) (ho : Principal (· + ·) o) :
```
then you can write `.of_principal_add` below

Reply:
I don't want to introduce any new dot notation for `IsLimit`, as it will all break when I deprecate it in favor of `IsSuccLimit` (hopefully quite soon now!)